src/lib/actions/ConfigAction.ts

```diff
@@ -41,6 +41,7 @@ export class ConfigAction {
 
 		// We always add a Logger Listener to the appropriate listeners list, because we should always be logging.
 		const logFileWriter: LogFileWriter = await LogFileWriter.fromConfig(userConfig);
+		this.dependencies.actionSummaryViewer.viewPreExecutionSummary(logFileWriter.getLogDestination());
```

Comment:
This is the functional change to this Action. I added it as early in the method as I could (i.e., immediately after the log file writer is instantiated) to minimize the possibility of it being preempted by an error.
If we want, though, we could easily move it further down and use it in the future to log additional information about, for example, the engines that are added. I'm flexible on this.